### ISSUE TITLE
All ModbusTCP plugins don't update to a changed IP on the fly

### ISSUE THREAD

**Issue by mzanetti** [nymea/nymea-plugins-modbus#82]
None of the ModbusTCP generator based plugins handle the case that a device changes IP address on the fly. They will stay disconnected until nymea is restarted.

**Comment by t-mon** [comment]
Also where the monitor has been introduced already?

**Comment by mzanetti** [comment]
Yes. Phoneixconnect seems to be the only one that is fixed already.

**Comment by mzanetti** [comment]
The reason is that the generated ModbusTcpConnection classes take the IP in the ctor, so it's only set at startup. Anyhow, luckily the base class has a setAddress() method. It seems to be enough to connect the monitors networkInfoChanged signal to that one (and the already existing reconnect timer will then take care about calling connectToHost() again).

**Comment by t-mon** [comment]
https://github.com/nymea/nymea-plugins-modbus/blob/master/libnymea-modbus/modbustcpmaster.cpp#L119

If reachable changes in the monitor, i perform a reconnect after setting the IP in the base class, which triggers the connectDevice() method where all properties get updated before actually performing the connection. 

For example: https://github.com/nymea/nymea-plugins-modbus/blob/master/kostal/integrationpluginkostal.cpp#L322

Not getting what the phoenix connect is doing different?

**Comment by mzanetti** [comment]
ok. right. fine for Kostal. Seems to be missing in alphainnotec and stiebeleltron now only....

**Comment by t-mon** [comment]
yeah, we cannot test those currently...

**Comment by t-mon** [comment]
Done with 1.11.0
